cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -961,7 +961,7 @@
     <name>metadata.updates.kafka.broker.list</name>
     <value>127.0.0.1:${kafka.bind.port}</value>
     <description>
-      CDAP Kafka service broker list to which metadata update notifications are published
+      Kafka service broker list to which metadata update notifications are published
```

Comment:
because we don't refer to Kafka generally as 'Kafka service'. The CDAP kafka is referred to as 'CDAP Kafka service'.

Also, CDAP Kafka is not different from Apache Kafka, its just an instance of Kafka that CDAP starts by default, and uses internally for logs, metrics and notifications. When referring to it generically, we should call it 'Apache Kafka' at least in the first occurrence.